### DISCUSSION TITLE
Strawberry perl 5.40.0.1 dev release

### DISCUSSION THREAD

**Discussion by shawnlaffan** [StrawberryPerl/Perl-Dist-Strawberry#199] [General]
I've cut a dev release of Strawberry Perl 5.40.0.1.  MSI, portable and PDL versions are available.  

Please test it.  

If there are no issues reported in the next week then I will upgrade it to a full release.  

https://github.com/StrawberryPerl/Perl-Dist-Strawberry/releases/tag/dev_5.40.0.1_20240803_gcc13b

**Comment by TheOncomingStorm57** [comment]
Greetings,

I enjoy using Strawberry Perl scripts for Oracle database evaluations.
Unfortunately, DBD support has been missing since strawberry-perl-5.32.1.1.
Is it correct that it will not return with strawberry-perl-5.40.0.1 either?
Will it be gone permanently?
I have posted the same question as a discussion because, unfortunately, I only saw this post now. Sorry for that..

**Comment by shawnlaffan** [comment]
SP 5.40.0.1 has been released so I'll close this topic.  

The Oracle discussion is under #216 (it warrants its own discussion)
